engine/rpcserver.go

```diff
@@ -2446,42 +2446,28 @@ func (s *RPCServer) GetHistoricCandles(ctx context.Context, r *gctrpc.GetHistori
 		End:      r.End,
 	}
 
-	var klineItem kline.Item
+	var klineItem *kline.Item
```

Comment:
I can't say where yet, but I think potentially with my latest time fix, there's something funky going on with candle times:
```
"start": "2022-12-24 11:00:00 AEDT",
 "end": "2023-01-03 11:00:00 AEDT",
 "interval": "24h",
 "candle": [
  {
   "time": "2022-12-24 16:00:00 UTC",
```

Start end are in local time which matched my expectations, but candle times are in UTC. I can (somewhat) tolerate that, but I would expect the first candle to be -11 hours to match with the start time, but there is a bigger disparity of times. So it looks like the request is not matching mine

For reference, my request is `go run . gethistoriccandles okx btc-usdt spot`